extensions/adobe/experience/campaign/feedbackevent.schema.json

```diff
@@ -0,0 +1,244 @@
+                  "description": "Address used as a to/callee address. Depending on the communication channel, the `address` has a different format.\n\n  * `email`: an email address.\n  * `textMessage` or `phone`: a shortcode or phone number."
+                },
+                "xdm:type": {
+                  "title": "Type",
+                  "type": "string",
+                  "description": "Whether this message message event describes the BCC, CC or To recipient."
+                }
+              }
+            }
+          }
+        },
+        "https://ns.adobe.com/experience/campaign/messageFailure": {
+          "title": "Message Failure",
+          "type": "object",
+          "properties": {
+            "xdm:canonicalReason": {
+              "title": "Canonical Reason",
+              "type": "string",
+              "description": "Canonicalized text of the response returned by the remote server due to a failed delivery attempt."
+            },
+            "xdm:reason": {
+              "title": "Reason",
+              "type": "string",
+              "description": "Unmodified, exact response returned by the remote server due to a failed delivery attempt."
+            },
+            "xdm:errorCode": {
+              "title": "Error Code",
+              "type": "string",
+              "description": "Error code by which the remote server described a failed delivery attempt."
+            },
+            "xdm:bounceType": {
+              "title": "Bounce Type",
+              "type": "string",
+              "description": "Classifies whether it is a soft bounce or hard bounce."
+            },
+            "xdm:bounceClass": {
+              "title": "Bounce Class",
+              "type": "string",
+              "description": "Classification code for a given message."
+            }
+          }
+        },
+        "https://ns.adobe.com/experience/campaign/solutionContext": {
+          "title": "Solution Context",
+          "type": "array",
+          "description": "List of solution specific key value pairs.",
+          "items": {
+            "$ref": "https://ns.adobe.com/experience/analytics/keyvalue"
+          }
+        },
+        "https://ns.adobe.com/experience/campaign/channel": {
+          "title": "Delivery channel",
+          "type": "object",
+          "properties": {
+            "xdm:type": {
+              "title": "Channel Type",
+              "type": "string",
+              "description": "Classifies whether the channel is email, push, sms."
+            },
+            "https://ns.adobe.com/experience/campaign/email": {
+              "title": "E-mail Channel",
+              "type": "object",
+              "properties": {
+                "https://ns.adobe.com/experience/campaign/interactions": {
+                  "title": "User Interactions",
+                  "type": "object",
+                  "properties": {
+                    "xdm:fbType": {
```

Comment:
What is fbType!

Reply:
This field is specific to the email channel. If the email has been reported as spam, then what type of spam is it. E.g. abuse, fraud. Ref: https://tools.ietf.org/html/rfc5965#section-7.3.